### PR TITLE
Update dependencies to latest versions, except angular

### PR DESCRIPTION
**Pull request by fwielstra**

### CODE REVIEW

**Comment by fvanwijk** [comment]
Don't use caret versions. It might break your build if someone releases a patch version with breaking change.

File: package.json [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
-    "grunt-ngmin": "0.0.2",
+    "grunt-ngmin": "0.0.3",
     "grunt-rev": "0.1.0",
     "jasmine-core": "2.5.2",
     "karma": "1.3.0",
     "karma-jasmine": "1.0.2",
-    "karma-phantomjs-launcher": "1.0.2"
+    "karma-phantomjs-launcher": "1.0.2",
+    "phantomjs": "^2.1.7",
+    "underscore": "^1.8.3"
```
Comment (fvanwijk) [inline package.json]: Lodash?

Comment (fvanwijk) [inline package.json]: You could remove `grunt-karma` because `npm test`

Comment (fvanwijk) [inline package.json]: You could remove `grunt-karma` because `npm test`

Comment (fwielstra) [inline package.json]: I should get rid of Grunt altogether, I don't think it's the right approach to have a library supply a minified version of itself anymore in this day and age; modern front-end apps will throw all of their shit together and optimize on the whole package.

As for lodash / underscore, I'd rather get rid of that too; it makes no sense for a 10 kb file (probably like 2 if you disregard the comments) to have that big a dependency.

**Comment by fwielstra** [comment]
As for caret functions, I agree; on the other hand, nobody should be releasing a patch version with a breaking change. NPM package maintainers can't be trusted though.

**Pull request merged**